projects/exui-common-lib/src/lib/components/hmcts-dialog/hmcts-dialog.component.scss

```diff
@@ -0,0 +1,192 @@
+.gem-c-modal-dialogue {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  outline: 0;
+}
+
+.gem-c-modal-dialogue__box {
+  display: block;
+  position: fixed;
+  background: #ffffff;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  overflow-y: auto;
+  border: 0;
+  font-family: "GDS Transport", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+}
+
+.gem-c-modal-dialogue__box:focus {
+  outline: 3px solid #ffdd00;
+}
+
+@media print {
+  .gem-c-modal-dialogue__box {
+    font-family: sans-serif;
+  }
+}
+@media (min-width: 40.0625em) {
+  .gem-c-modal-dialogue__box {
+    font-size: 1.1875rem;
+    line-height: 1.3157894737;
+  }
+}
+@media print {
+  .gem-c-modal-dialogue__box {
+    font-size: 14pt;
+    line-height: 1.15;
+  }
+}
+@media print {
+  .gem-c-modal-dialogue__box {
+    color: #000000;
+  }
+}
+@media (min-width: 40.0625em) {
+  .gem-c-modal-dialogue__box {
```

Comment:
This selector (or the one further up) is the one you use to target what element becomes visible as a dialog, so that element is defined in the implementing app, not here. Having styles for that element here will either not work because of scoping, or cause clashes with styles defined in the implementing app. Neither is desirable.

Reply:
this will eventually come from hmcts/front-end once they promote it to there